common/entrypoint_mautic_cron.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ ! -f /opt/mautic/cron/mautic ]; then
+
+	cat <<EOF > /opt/mautic/cron/mautic
```

Comment:
/entrypoint_mautic_cron.sh: line 5: /opt/mautic/cron/mautic: No such file or directory

This is what I got when I'm running this entrypoint

Reply:
I was missing volume configuraion for this one to work. On the other hand it I believe it should work without volume attached as well.

Reply:
@lumnn good point! I'll ensure the dir exists

Reply:
@lumnn see https://github.com/mautic/docker-mautic/pull/238/commits/eb3ce519f423e256d534a7ac40c52a34df6cb2b3